src/azure-cli/azure/cli/command_modules/batch/_help.py

```diff
@@ -225,6 +225,37 @@
 short-summary: List virtual machine SKUs available in a location.
 """
 
+helps['batch private-link-resource'] = """
+type: group
+short-summary: Manage Batch account private Link Resources.
+"""
+
+helps['batch private-link-resource show'] = """
+type: command
+short-summary: Gets information about the specified private link resource.
```

Comment:
Please use the first person voice in the help message
```suggestion
short-summary: Get information about the specified private link resource.
```